docs/sdk/src/reference_docs/chain_spec_runtime/Cargo.toml

```
@@ -42,6 +42,7 @@ substrate-wasm-builder = { optional = true, workspace = true, default-features =
 
 [dev-dependencies]
 chain-spec-builder = { workspace = true, default-features = true }
+cmd_lib.workspace = true
```

nit: would be great to use the same pattern as the one used for the other
```suggestion
cmd_lib = { workspace = true }
```